### ISSUE TITLE
Browser build has started failing; possibly Acorn 1.0.3 -> 1.1.0 related

### ISSUE THREAD

**Issue by domenic** [tmpvar/jsdom#1131]
https://travis-ci.org/tmpvar/jsdom/jobs/62017500#L186 passes, installed Acorn 1.0.3

https://travis-ci.org/tmpvar/jsdom/jobs/62578383 fails, installed Acorn 1.1.0

Failures are related to global variables inside workers:

```
running jsdom/inside-worker-smoke-tests jsdom/inside-worker-smoke-tests

✖ jsdom/inside-worker-smoke-tests/execute scripts with global variables / window scope reference

strictEqual: undefined === "true"

nested reference should work

✖ jsdom/inside-worker-smoke-tests/test async global variable context

strictEqual: undefined === "true"

strictEqual: undefined === "baz"
```

No obvious culprits in the Acorn commit log though so unsure.

Probably easy to debug if someone takes the time to spin up a browser build.

**Comment by domenic** [comment]
Confirmed it's Acorn 1.1.0.

**Comment by domenic** [comment]
Fix in Acorn 1.2.1, which is automatically picked up. I'll bump the minimum version but no need to publish anew I think.
